samples/client/petstore/cpp-qt5/client/OAIPetApi.cpp

```diff
@@ -91,7 +91,7 @@ OAIPetApi::deletePet(const qint64& pet_id, const QString& api_key) {
     OAIHttpRequestInput input(fullPath, "DELETE");
 
     if (api_key != nullptr) {
-        input.headers.insert("api_key", "api_key");
+        input.headers.insert("api_key", api_key);
```

Comment:
this change is not part of this PR. Were the samples not regenerated in the PR that performed this change?

Reply:
No it was not updated. But this is correct.